curriculum/challenges/_meta/learn-html-by-building-a-cat-photo-app/meta.json

```diff
@@ -277,5 +277,10 @@
     [
       "5ef9b03c81a63668521804ee",
       "Step 67"
+    ],
+    [
+      "62bb4009e3458a128ff57d5d",
+      "Step 68"
     ]
-  ]}
+  ]
+}
```

Comment:
What's the convention about this in the fCC codebase? Most files seem to have that extra newline at the end